pubsub/azure/servicebus/servicebus.go

```diff
@@ -489,7 +487,6 @@ func (a *azureServiceBus) Subscribe(req pubsub.SubscribeRequest, handler pubsub.
 			attempts := readAttemptsStale()
 			if attempts == 0 {
 				a.logger.Errorf("Subscription to topic %s lost connection, unable to recover after %d attempts", sub.topic, a.metadata.MaxReconnectionAttempts)
```

Comment:
This is unchanged, but wondering if we should make the app crash here (`Fatalf`)? Otherwise, the error would get logged and the app continues to work but without a connection to Service Bus.